ptz-explainer.md

```diff
@@ -107,12 +100,11 @@ if ("tilt" in capabilities) {
 ```
 
 The example below shows how camera pan could be reset when acquiring a
-PTZ camera in `getUserMedia()`.
+PTZ camera in `getUserMedia()`. Only ideal constraints are allowed for pan,
+tilt, and zoom constraints. Using mandatory ones will cause the returned promise
+to reject with `OverConstrainedError`.
 
 ```js
-// User is prompted to grant both camera and PTZ access in a single call.
-// If the camera does not support PTZ or user denies PTZ permission, it fails
-// as PTZ constraints are required.
 const videoStream = await navigator.mediaDevices.getUserMedia({
   // [NEW] Website asks to reset camera pan.
   video: { pan: 1 },
```

Comment:
Why 1 and not 0?

Reply:
You're right. It will mostly be `0`, not `1`

Reply:
Fixed in https://github.com/w3c/mediacapture-image/pull/258/commits/05174085a0f6affc5b3b3cb6bb27b01ca611e19a. Thanks!